ServiceFinder/ServiceFinder.API/Exceptions/FluentValidatorException.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace ServiceFinder.API.Exceptions
+{
+
+    public class FluentValidatorException : Exception
+    {
+        public int StatusCode { get; } = 400;
+
+        public FluentValidatorException() { }
+
+        public FluentValidatorException(string message, Exception innerException) : base(message, innerException) { }
+
+        public FluentValidatorException(string message) : base(message) { }
+    }
+}
```

Comment:
Fluent Validator has its own implemented exception. Why have you implemented this logic on your side?